Packages/vcs/Lib/VTKPlots.py

```diff
@@ -357,7 +358,7 @@ def createRenderer(self, *args, **kargs):
         return ren
 
     def update(self, *args, **kargs):
-        self._lastSize = -1
+        self._lastSize = None
```

Comment:
@doutriaux1 No, I haven't. I have my mac with me and I can run the tests. I tried to be very conservative in what I change. :-)